### DISCUSSION TITLE
Is the window supposed to go black when resizing?

### DISCUSSION THREAD

**Discussion by dennisorlando** [bevyengine/bevy#8580] [Q&A]
Hi, I was wondering if this was a bug or something that I could fix. I can't get any info on the internet about this problem whatsoever, therefore I'm not sure how to even start tackling it.
I previously used the Speedy2d library and I recall having some similar problems with it, but now I just tried their "animation" example and it worked flawlessly

[Screencast from 2023-05-09 17-26-36.webm](https://github.com/bevyengine/bevy/assets/47061464/bff16c6e-6be5-40f3-9e2a-e33a6665f517)

The same problem occurs with bevy's official examples (I tried "2d_shapes" and "pbr"), which is why I'm not including any code

**Comment by dennisorlando** [comment]
Update: it might be worth mentioning that I'm running ArchLinux on a laptop with switchable graphics (either dedicated or optimus):
- when the screen is hooked to the iGPU (window runs on nvidia then gets sent to iGPU for display rendering), I get the "black window" that appears in the first video. The same exact behavior appears when running "glxgears" 
- when hooking the screen directly to the dGPU (rtx 3060, using MUX switch), I get this instead:
[Screencast from 2023-05-09 17-50-14.webm](https://github.com/bevyengine/bevy/assets/47061464/a6244245-4ba0-4dca-b307-1321e3b37c7f)

**Comment by dennisorlando** [comment]
Update: when turning off the dgpu completely (therefore only using intel iGPU), then glxgears gets the problem (completely black when resizing) while the bevy app works like a charm, smooth and without any particular problem. 
During all these tests, the other apps worked perfectly fine (firefox, nautilus, clion, alacritty);

I guess I'll file a bug report unless someone has some insight on this

**Comment by dimvoly** [comment]
FWIW on Ubuntu 20.04 for me when resizing quickly I only get a small bit of black when making the window bigger but only for a split second. So I don't think you're supposed to get a black window like you describe.
